es/es.go

```diff
@@ -265,6 +274,36 @@ func Run(ctx app.Context) {
 	etre.Debug("trace: %s", trace)
 	ec = ec.WithTrace(trace)
 
+	// //////////////////////////////////////////////////////////////////////
+	// Insert and exit, if --insert
+	// //////////////////////////////////////////////////////////////////////
+
+	if o.Insert {
+		// cmdLine.Args validated above
+		ctx.Patches = cmdLine.Args[1:]
+
+		if ctx.Hooks.BeforeInsert != nil {
+			etre.Debug("calling hook BeforeInsert")
+			err = ctx.Hooks.BeforeInsert(&ctx)
+			if err != nil {
+				printAndExit(fmt.Errorf("BeforeInsert hook failed: %s", err), ctx)
+			}
+		}
+
+		patch, err := parsePatches(ctx)
+		if err != nil {
+			printAndExit(err, ctx)
+		}
+
+		wr, err := ec.Insert([]etre.Entity{patch})
+		_, err = writeResult(ctx, set, wr, err, "insert")
+		if err != nil {
+			printAndExit(err, ctx)
+		}
+		fmt.Printf("OK, inserted %s %s\n", ctx.EntityType, wr.Writes[0].EntityId)
```

Comment:
Maybe this case doesn't exist, but just to be safe
```suggestion
if len(wr.Writes) != 1 {
   printAndExit(fmt.Errorf("Etre reports no error but %d insert, expected 1 insert: %v", len(wr.Writes)), ctx)
}

		fmt.Printf("OK, inserted %s %s\n", ctx.EntityType, wr.Writes[0].EntityId)
```